plugins/review_plugin.py

```diff
@@ -0,0 +1,81 @@
+import re
+import hashlib
+
+from slackbot.bot import respond_to
+
+from common.utils import render, make_api_request
+
+
+class ReviewHelper:
+
+    @staticmethod
+    def make_api_request_for_review(serving_id, rating_value, anonymity_id, comment=''):
+        query = 'mutation{createReview(input: {serving: "%s", value: "%s", \
+                anonymityId: "%s", comment: "%s"})\
+                {review{id,originalId,value,comment,anonymityId,serving{publicId}}}}' % \
+                (serving_id, rating_value, anonymity_id, comment)
+        res = make_api_request(query)
+
+        if 'review' in res:
+            if res['review'] is None:
+                return 'duplicate'
```

Comment:
It's not only duplicate will result to `None` value. I think we should return generic message here